cub/cub/device/dispatch/dispatch_histogram.cuh

```diff
@@ -234,12 +236,12 @@ __launch_bounds__(int(ChainedPolicyT::ActivePolicy::AgentHistogramPolicyT::BLOCK
   AgentHistogramT agent(
     temp_storage,
     d_samples,
-    num_output_bins_wrapper.array,
-    num_privatized_bins_wrapper.array,
-    d_output_histograms_wrapper.array,
-    d_privatized_histograms_wrapper.array,
-    output_decode_op_wrapper.array,
-    privatized_decode_op_wrapper.array);
+    num_output_bins_wrapper.begin(),
+    num_privatized_bins_wrapper.begin(),
+    d_output_histograms_wrapper.begin(),
+    d_privatized_histograms_wrapper.begin(),
+    output_decode_op_wrapper.begin(),
+    privatized_decode_op_wrapper.begin());
```

Comment:
@miscco I could make my life a lot easier (and retain `AgentHistogramT`), if I could just pull the underlying array out, like:
```suggestion
    num_output_bins_wrapper.__elems_,
    num_privatized_bins_wrapper.__elems_,
    d_output_histograms_wrapper.__elems_,
    d_privatized_histograms_wrapper.__elems_,
    output_decode_op_wrapper.__elems_,
    privatized_decode_op_wrapper.__elems_);
```
What's your degree of unhappiness with such reliance on our stdlib internals?

Reply:
Honestly, I believe we can make our lives easier. 

It is our type at last

Reply:
Thx. Applied.